src/main/java/com/questhelper/quests/beneathcursedsands/BeneathCursedSands.java

```diff
@@ -0,0 +1,591 @@
+
+		ConditionalStep returnToZahur = new ConditionalStep(this, pickLilyOfElid);
+		returnToZahur.addStep(new Conditions(lilyOfTheElid), takeLilyToZahur);
+		steps.put(76, returnToZahur);
+		steps.put(78, returnToZahur);
+
+		steps.put(80, talkToZahur);
+		steps.put(82, warmUpChemistryEquipment);
+		steps.put(84, talkToZahur);
+		steps.put(86, talkToZahur);
+
+		steps.put(88, bringCureToPriest);
+		steps.put(90, talkToSophanemHighPriest);
+		steps.put(92, prepareFightMenaphiteAkh);
+		steps.put(94, prepareFightMenaphiteAkh);
+		steps.put(96, prepareFightMenaphiteAkh);
+
+		ConditionalStep menaphiteAkhFight = new ConditionalStep(this, defeatMenaphiteAkh);
+		menaphiteAkhFight.addStep(shouldFightMenaphiteShadow, defeatMenaphiteShadow);
+		steps.put(98, menaphiteAkhFight);
+		steps.put(100, talkToOsman);
+		steps.put(102, finishQuest);
+		steps.put(104, finishQuest);
+		steps.put(106, finishQuest);
+
+		return steps;
+	}
+
+	public void setupItemRequirements()
+	{
+		coal = new ItemRequirement("Coal", ItemID.COAL);
+		tinderbox = new ItemRequirement("Tinderbox", ItemID.TINDERBOX);
+		tinderbox.setTooltip("Obtainable during quest");
+		ironBar = new ItemRequirement("Iron bar", ItemID.IRON_BAR);
+		spade = new ItemRequirement("Spade", ItemID.SPADE);
+		spade.setTooltip("Obtainable during quest");
+		meat = new ItemRequirement("Any cooked or raw meat", ItemID.COOKED_MEAT);
+		meat.addAlternates(ItemID.RAW_BEEF, ItemID.RAW_BEAR_MEAT, ItemID.RAW_BOAR_MEAT, ItemID.RAW_RAT_MEAT, ItemID.RAW_YAK_MEAT);
+		meat.setTooltip("Purchasable from a shop during the quest. Fish will NOT work");
+		waterskins = new ItemRequirement("Waterskin(s)", ItemID.WATERSKIN4);
+		waterskins.addAlternates(ItemID.WATERSKIN3, ItemID.WATERSKIN2, ItemID.WATERSKIN1);
+		waterskins.setTooltip("Used for protection against the desert heat");
+		antipoison = new ItemRequirement("Antipoison", ItemCollections.getAntipoisons());
+		accessToFairyRings = new ItemRequirement("Access to Fairy Rings", ItemID.DRAMEN_STAFF);
+		accessToFairyRings.addAlternates(ItemID.LUNAR_STAFF);
+		pharaosSceptre = new ItemRequirement("Pharaoh's sceptre", ItemCollections.getPharoahSceptre());
+		pharaosSceptre.setTooltip("When visiting Necropolis during the quest, you can unlock the direct teleport by using 'Commune' on the Obelisk.");
+		food = new ItemRequirement("Food", -1, -1);
+		food.setDisplayItemId(BankSlotIcons.getFood());
+		combatGear = new ItemRequirement("Combat gear", -1, -1);
+		combatGear.setDisplayItemId(BankSlotIcons.getCombatGear());
+		combatGearChampionOfScabaras = new ItemRequirement("Ranged combat gear", -1, -1);
+		combatGearChampionOfScabaras.setDisplayItemId(BankSlotIcons.getRangedCombatGear());
+		staminaPotions = new ItemRequirement("Stamina Potions", ItemCollections.getStaminaPotions(), -1);
+		nardahTeleport = new ItemRequirement("Nardah Teleport", ItemID.NARDAH_TELEPORT);
+		nardahTeleport.addAlternates(ItemID.DESERT_AMULET_2, ItemID.DESERT_AMULET_3, ItemID.DESERT_AMULET_4);
```

Comment:
Hmm, I get swapping it, but why prefer the Nardah tele over the lower amulets exactly? The amulets themselves all have the same functionality as far as I'm aware, but since they're Diary unlocks they are differently tiered.

Reply:
Nevermind, I was mistaken. Lower tiers are limited in charges. Updating.